evergreen/config.yml

```diff
@@ -1194,6 +1201,21 @@ task_groups:
   - .object_store_test_suite
   - package
 
+# Runs core/sync/local object store tests
```

Comment:
This is fine. But FYI we now have a "CombinedTests" build target that runs core/sync/object-store tests in a single binary.